ssl/src/main/java/org/wildfly/security/ssl/SSLContextBuilder.java

```diff
@@ -60,6 +60,8 @@
 public final class SSLContextBuilder {
 
     private SecurityDomain securityDomain;
+    // This should be changed to CipherSuiteSelector.openSslCombinedDefault once we are ready to enable
```

Comment:
I think there is an ELY issue to enable TLS 1.3 by default?  Maybe reference the issue in the comments so we can find them easily?

Reply:
I've now referenced the ELY issue.